src/sentry/plugins/bases/notify.py

```diff
@@ -66,7 +66,7 @@ def get_plugin_type(self):
 
     def notify(self, notification):
         event = notification.event
-        return self.notify_users(event.group, event)
+        return self.notify_users(event.group, event, map(lambda r: r.label, notification.rules))
```

Comment:
Could use a list comprehension here

```suggestion
        return self.notify_users(event.group, event, [r.label for r in notification.rules])
```